code/style.css

```diff
@@ -1,4 +1,189 @@
+    to right,
+    #a3d2ca,
+    #5eaaa8
+  ); /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(
+    to right,
+    #a3d2ca,
+    #5eaaa8
+  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+/*** HEADER ***/
+header {
+  background-color: #e8ded2;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0px;
+}
+
+.site-logo-name {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-left: 50px;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.logo {
+  width: 100px;
+  height: 100px;
+}
+
+nav {
+  display: none;
+}
+
+nav a:hover {
+  font-size: 24px;
+  font-weight: bold;
+  border-top: 2px solid #056676;
+  border-bottom: 2px solid #056676;
+}
+
+/**** BIG NEWS ****/
+.big-news-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 40px;
+}
+
+.big-news-card {
+  margin: 10px 40px;
+  width: 80%;
+}
+
+.big-news-card img {
+  width: 100%;
+  border-radius: 30px;
+}
+
+.big-news-card h2 {
+  text-align: center;
+}
+
+/**** OTHER NEWS ****/
+.other-news-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 40px;
+}
+
+.other-news-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #e8ded2;
+  margin: 10px 40px;
+  border-radius: 30px;
+  box-shadow: 5px 5px #868686;
+  width: 80%;
+  height: auto;
+}
+
+.other-news-card img {
+  width: 100%;
+  border-top-left-radius: 30px;
+  border-top-right-radius: 30px;
+}
+
+/*** FOOTER ***/ /* If I have less other-news-card the footer will not be at the bottom - TODO fixed footer at the bottom of the page (all sizes)  */
+footer {
+  background-color: #056676;
+  color: #e8ded2;
+  text-align: center;
+  padding: 20px 10px;
+}
+
+footer a {
+  color: #e8ded2;
+}
+
+/*** MEDIA QUERIES ***/
+@media (min-width: 768px) {
+  .site-logo-name {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+  }
+
+  nav a {
+    color: #056676;
+    text-decoration: none;
+  }
+
+  .big-news-container {
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .big-news-card {
+    width: 40%;
+  }
+
+  .other-news-container {
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .other-news-card {
+    width: 30%;
+  }
+}
+
+@media (min-width: 1024px) {
+  header {
+    justify-content: space-between;
+  }
+
+  .site-logo-name {
+    display: flex;
+  }
+
+  nav {
+    width: 50%;
+  }
+
+  nav .logo {
+    display: none;
+  }
+
+  .big-news-card {
+    width: 30%;
+  }
+
+  .big-news-card-grow {
+    flex-grow: 2;
+  }
+
+  .other-news-container {
+    justify-content: flex-start; /* What do I need to do to get them to be center but still get the new row left aligned (not spread out)? */
```

Comment:
`justify-content: center;` ? :)

Reply:
Take a look at this stack overflow question as well: [https://stackoverflow.com/questions/43208025/center-a-group-of-wrapped-flex-items](https://stackoverflow.com/questions/43208025/center-a-group-of-wrapped-flex-items)